### DISCUSSION TITLE
Personal Views ETA

### DISCUSSION THREAD

**Discussion by kudryk-enverus** [nocodb/nocodb#4860] [General]
My team is new to NocoDB. We were intending on deploying it so that multiple users could review and update a dataset, but then I began to encounter one user's filters were showing up in my session. This is a known issue that Stephanof [raised last year](https://github.com/nocodb/nocodb/discussions/3068).

I also see in the documentation that the solution to this is [Personal Views](https://docs.nocodb.com/setup-and-usages/views/#personal-views), but which is not currently available but will be in a future release. As it's mentioned in the docs, is there an ETA for when this will be released?

**Comment by qtld88** [comment]
+1 !
We would love to have an ETA to know if we have to find another software or if we can start working on this one knowing this crucial feature will come really soon !
Thanks !
